com.unity.multiplayer.mlapi/Prototyping/NetworkAnimator.cs

```diff
@@ -247,7 +247,7 @@ public enum Authority
 
         [SerializeField]
         private float m_SendRate = 0.1f;
```

Comment:
should we also kill this too? I asked the same question before in @pdeschain's `NetworkAnimator` PR. Do we really want to control `NetworkAnimator`'s tickrate separately from network config's tickrate? If so, why? If not, why keep `m_SendRate` and its implementation around?

Reply:
Most likely yes, we will want to remove this. But I don't think it should be in this PR and I think we first need to better understand how we will do NetworkVariables / priority in the future.